### ISSUE TITLE
Support using system CUDA on Ubuntu

### ISSUE THREAD

**Issue by pitrou** [numba/numba#3738]
When using the Numba wheel, it cannot find the Ubuntu-installed CUDA toolkit by default.
I had to do the following to get a trivial kernel to work:
```python
>>> os.environ['NUMBAPRO_NVVM']= '/usr/lib/x86_64-linux-gnu/libnvvm.so'
>>> os.environ['NUMBAPRO_LIBDEVICE']= '/usr/lib/cuda/nvvm/libdevice'
```

It would be nice if this was detected automatically ;-)

**Comment by seibert** [comment]
Agreed.  We should fall through to the standard system locations if nothing is found in the current environment.

**Comment by ccoulombe** [comment]
#4059 solves this.

**Comment by gmarkall** [comment]
Closing as I believe this is resolved by #5016
